chain/indexer/README.md

```diff
@@ -0,0 +1,62 @@
+# NEAR Indexer
+
+NEAR Indexer is a micro-framework, which provides you with a stream of blocks that are recorded on NEAR network.
+
+
+NEAR Indexer is useful to handle real-time "events" on the chain.
+
+
+NEAR Indexer is going to be used to build NEAR Explorer, augment NEAR Wallet, and provide overview of events in Rainbow Bridge.
+
+
+See the [example](https://github.com/nearprotocol/nearcore/tree/master/tools/indexer/example) for further details.
+
+
+## How to set up and test NEAR Indexer
+
+Assuming you have all necessary tools installed (Rust, Cargo etc.)
```

Comment:
@khorolets how does one know which tools are necessary if even we cannot state them explicitly? :smile: 

```suggestion
Before you proceed, make sure you have the following software installed:

* [rustup](https://rustup.rs/) or Rust version that is mentioned in `rust-toolchain` file in the root of nearcore project.
```